Formula/imagemagick.rb

```diff
@@ -132,6 +136,14 @@ def caveats
   end
 
   test do
-    system "#{bin}/identify", test_fixtures("test.png")
+    system bin/"identify", test_fixtures("test.png")
+    # Test recommended features and delegates.
+    features = shell_output("#{bin}/convert -version")
+    assert_match "freetype", features if build.with? "freetype"
```

Comment:
Do we care about testing builds with options? Quite a while ago I was told that we don't.